docs/configuration.asciidoc

```diff
@@ -280,7 +280,7 @@ This config value allows you to specify which cloud provider should be assumed
 for metadata collection. By default, the agent will attempt to detect the cloud
 provider or, if that fails, will use trial and error to collect the metadata.
 
-Valid options are `"aws"`, `"gcp"`, and `"azure"`. If this config value is set
+Valid options are `auto`, `"aws"`, `"gcp"`, and `"azure"`. If this config value is set
```

Comment:
Unless I'm missing something, could you wrap `auto` in quotes analogue to the other options for consistency?